bahmnicore-api/pom.xml

```diff
@@ -197,6 +197,11 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>emrapi-api-1.12</artifactId>
+            <scope>compile</scope>
```

Comment:
why? emrapi is bundled already with distro. https://github.com/Bahmni/bahmni-core/blob/OMRS-master-2.4/bahmnicore-omod/src/main/resources/config.xml#L25

Reply:
yes @angshu I tired removing the scope as compile time but few of the IT test cases are failing for not being able to find the classes from emrapi-api-1.12. and sometimes because of the hiberanteCache issue. And the error reasons are inconsistent.
**Caused by: net.sf.ehcache.CacheException: Another CacheManager with same name 'hibernateCache' already exists in the same VM. Please provide unique names for each CacheManager in the config or do one of following:
1. Use one of the CacheManager.create() static factory methods to reuse same CacheManager with same name or create one if necessary
2. Shutdown the earlier cacheManager before creating new one with same name.**